command/do_transition.go

```diff
@@ -0,0 +1,72 @@
+package command
+
+import (
+	"encoding/json"
+	"github.com/ExpediaGroup/flyte-jira/client"
+	"github.com/HotelsDotCom/flyte-client/flyte"
+	"log"
+)
+
+var (
+	DoTransition = flyte.Command{
+		Name:         "DoTransition",
```

Comment:
As per other comment just use `Transition` all across the board drop `Do`

Reply:
done